Makefile

```
@@ -12,4 +12,4 @@ run: docker
 # when run with 'make run', makes sure that it responds correctly
 run-test:
 	curl -v 'http://localhost:8085/rate/submit.php?version=1.234&rating=0&issue=80386'
-	curl -v http://localhost:8085/rate/result.php 2>&1 | grep 80386
+	curl -v http://localhost:8085/rate/result.php 2>&1
```

Is there a reason to remove the grep (which would fail as expected if the searched token is absent)?

I would have suggested to add a `jq` command to ensure the returned content is a valid json (which would fail with the HTML returned on the current state)

Soemthing like this: (to adapt)=

```suggestion
        command -v jq || { echo 'ERROR: Command line "jq" not found.'; exit 1}'
	curl -v http://localhost:8085/rate/result.php | jq -r '<to be determined>.80386'
```

What

---

seems not so easy with the command check ... I will let that for later but keep the no grep to get the headers shown for now